src/IssueViz/Selection/AnalysisIssueNavigation.cs

```diff
@@ -22,36 +22,39 @@
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
 using System.Linq;
+using SonarLint.VisualStudio.IssueVisualization.Editor;
 using SonarLint.VisualStudio.IssueVisualization.Models;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Selection
 {
     internal interface IAnalysisIssueNavigation
     {
-        void GotoNextLocation();
-        void GotoPreviousLocation();
+        void GotoNextNavigableLocation();
```

Comment:
Renamed for better accuracy

Reply:
Could be `GotoNextNavigableStep` / `GotoNextNavigableFlowStep` to make it clearer that this is navigation with a single issue flow.